docs/proposals/crd-namespace-migration.md

```diff
@@ -0,0 +1,33 @@
+# CRD Group Migration to K8c.io
+
+**Author**: Mohamed Elsayed (@moelsayed)
+
+**Status**: Draft proposal;
+
+## Goals
+Kubernets API groups are organized by namespaces. Most of KPP CRDs are currently using the `k8s.io` namespace. 
+
+The `k8s.io` namespace is owned/managed by the Kubernets community and is currently protected by an API review process. Ath this point, the Kube-API implements checks to validate if a specific CRD is allowed to use this namespace or not. Unfortunately, we are not allowed. 
+
+This proposal plans to migrate all KPP CRDs from the `k8s.io` namespace into the `kubermatic.io` namespace. 
```

Comment:
Let's discuss this. What about `k8c.io`? Short, but maybe confusing (w.r.t. `k8s.io`)?

Reply:
I actually wrote `k8c.io` here first, then changed it to `kubermatic.io` when I noticed your note about having that already for the operator. My +1 is actually for `k8c.io`

Reply:
Apparently we had already discussed this previously and settled on `k8c.io`, according to my team.